README.md

```diff
@@ -1 +1,139 @@
+```
+$ git clone git@github.com:ha-manel/Luxy-Rides-API.git
+```
+### Usage
+
+Kindly modify the files as needed.
+
+### Run locally
+
+Make your way to the correct directory by running this command:
+
+```
+$ cd Luxy-Rides-API
+```
+
+Install the required dependencies to run the project with this command:
+```
+$ bundle install
+```
+
+Setup the database, run:
+```
+$ rails db:create
+$ rails db:migrate
+$ rails db:seed
+```
+
+Then run it in your browser with this command:
+
+```
+$ rails server
+```
+
+### Run tests
+
+To run tests, please run this command:
+```
+$ rspec
+```
+## Authors
+
+👤 **Manel Hammouche** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- <img src="https://user-images.githubusercontent.com/67911212/185442918-aa30589c-c9f9-4edb-8955-1036ceebd5c2.png" width="18"/> [ha-manel](https://github.com/ha-manel)
+- <img src="https://user-images.githubusercontent.com/67911212/185441124-47527d95-39c5-4984-9d2c-a130be72bd50.png" width="18"/> [ha_manel_](https://twitter.com/ha_manel_)
+- <img src="https://user-images.githubusercontent.com/67911212/185442306-ef777855-06ac-4e36-b649-6f0dda869366.png" width="18"/> [Manel Hammouche](https://www.linkedin.com/in/manel-hammouche/)
+
+👤 **Toussaint Saraza** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- <img src="https://user-images.githubusercontent.com/67911212/185442918-aa30589c-c9f9-4edb-8955-1036ceebd5c2.png" width="18"/> [sarazaToussaint](https://github.com/sarazaToussaint)
+- <img src="https://user-images.githubusercontent.com/67911212/185441124-47527d95-39c5-4984-9d2c-a130be72bd50.png" width="18"/> [toussaintSaraz1](https://twitter.com/ToussaintSaraz1)
+- <img src="https://user-images.githubusercontent.com/67911212/185442306-ef777855-06ac-4e36-b649-6f0dda869366.png" width="18"/> [Toussaint Saraza](https://www.linkedin.com/in/toussaintsaraza/)
+
+👤 **Hammad Siddique** <img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="20"/>
+
+- <img src="https://user-images.githubusercontent.com/67911212/185442918-aa30589c-c9f9-4edb-8955-1036ceebd5c2.png" width="18"/> [HammadSiddique](https://github.com/HammadSiddique)
+- <img src="https://user-images.githubusercontent.com/67911212/185441124-47527d95-39c5-4984-9d2c-a130be72bd50.png" width="18"/> [hs_devv](https://twitter.com/hs_devv)
+- <img src="https://user-images.githubusercontent.com/67911212/185442306-ef777855-06ac-4e36-b649-6f0dda869366.png" width="18"/> [HammadSiddique](https://www.linkedin.com/in/-hammadsiddique/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+- [Microverse](https://www.microverse.org/)
+
+## 📝 License
```

Comment:
- [ ] Your project does not have a license file.  Please, kindly go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file, and add it to the root of your project.  After that, please include a link to your license file on your readme.

This is true for both the API and the frontend readmes.

That way, your project will be well documented 💪

Reply:
- [x] Done, thanks for pointing this out 👍